### DISCUSSION TITLE
play.tailwindcss.com  front-end source code

### DISCUSSION THREAD

**Discussion by 18355166248** [tailwindlabs/tailwindcss#11697] [Help]
May I ask if I would like to obtain https://play.tailwindcss.com Does the front-end source code have a warehouse address

**Wanna learn something**

**Comment by tobyab** [comment]
As far as I can see, the only repo that references a tailwind playground is here: https://github.com/tailwindlabs/tailwindcss-playground. However, this is now just a read-only repo.
